docs/extensibility/extending-blocks.md

```diff
@@ -155,20 +155,20 @@ wp.hooks.addFilter( 'editor.BlockEdit', 'my-plugin/with-inspector-controls', wit
 {% ESNext %}
 ```js
 const { createHigherOrderComponent } = wp.compose;
-const { Fragment } = wp.Element;
+const { Fragment } = wp.element;
 const { InspectorControls } = wp.editor;
 const { PanelBody } = wp.components;
 
 const withInspectorControls =  createHigherOrderComponent(BlockEdit => {
   return props => {
     return (
```

Comment:
It looks like we are using spaces instead of tabs in this lines. (already existing problem).